modules/admin_manual/pages/installation/ubuntu_18_04.adoc

```diff
@@ -160,6 +160,8 @@ occ config:system:set trusted_domains 1 --value="$myip"
 ----
 echo "*/15  *  *  *  * /usr/bin/php -f {install-directory}/cron.php" \
   > /var/spool/cron/crontabs/{webserver-user}
+echo "13 0  *  *  * /usr/bin/php -f {install-directory}/occ dav:cleanup-chunks" \
+  > /var/spool/cron/crontabs/{webserver-user}
```

Comment:
Should we document that, by default, the command cleans up chunks older than 2 days and that the number of days is actually configurable as input option?

Reply:
Sounds like a good idea. I'll get that done.

Reply:
Honestly, I'm not sure what that wasn't added, originally, as part of this PR. Adding it now. Thanks again, @pako81.

Reply:
@pako81, I just grokked the docs and found that the relevant information's already documented in `occ_command.adoc` and `background_jobs_configuration.adoc.`